sdk/openai/openai/test/public/utils/asserts.ts

```diff
@@ -180,6 +180,7 @@ function assertToolCall(
 ): void {
   assertIf(!stream, functionCall.type, assert.isString);
   assertIf(!stream, functionCall.id, assert.isString);
+  assertIf(!stream, functionCall.index, assert.isNumber);
```

Comment:
This should be returned in the streaming case, right?
```suggestion
  assertIf(stream, functionCall.index, assert.isNumber);
```